habitat/datasets/rearrange/generate_episode_inits.py

```diff
@@ -6,15 +6,17 @@
 
 import argparse
 
-from tqdm import trange
+from tqdm import tqdm
 
 import habitat
 
 
 def generate_inits(cfg_path, opts):
     config = habitat.get_config(cfg_path, opts)
     with habitat.Env(config=config) as env:
-        for _ in trange(range(env.number_of_episodes)):
+        for i in tqdm(range(env.number_of_episodes)):
+            if i % 100 == 0:
+                print(cfg_path, config.DATASET.DATA_PATH)
```

Comment:
What does this print statement do? Is cfg_path or config.DATASET.DATA_PATH supposed to change?

Reply:
No it does not change. It prints the dataset the initializiations are being generated for. This is useful when the job is running for a long time and you cannot see the job that is being run. Printing reminders the user what the job is for.